docs/server/source/quickstart.md

```diff
@@ -57,6 +57,13 @@ I. Run BigchainDB Server:
 $ bigchaindb start
 ```
 
+J. Verify BigchainDB Server setup by visiting the BigchainDB Root Url in your browser (if not changed during installation):
+```text
+$ http://127.0.0.1:9984/
```

Comment:
TL;DR: yep! it's ok :1st_place_medal: 

`127/8` (that is: `127.0.0.0` with subnet mask `255.0.0.0`) is reserved (see [rfc5735](https://tools.ietf.org/html/rfc5735#section-3)) so it's always the same for the loopback address. You can also put `127.123.111.42` if you want to confuse people :see_no_evil: